Update documentation (#40)
kyma-incubator/service-catalog: docs/devguide.md

    │   └── catalog             # Helm chart for deploying the service catalog
    │   └── ups-broker          # Helm chart for deploying the user-provided service broker
    ├── cmd                     # Contains "main" Go packages for each service catalog component binary
    │   └── controller-manager  # The service catalog controller manager service-catalog command
    │   └── service-catalog     # The service catalog binary, which is used to run commands
    │   └── svcat               # The command-line interface for interacting with kubernetes service-catalog resources

>     │   └── webhook             # The service catalog webhook server webhook command

Reviewer: Is it correct?

Author: I changed it to `The service catalog webhook server command`

for more details.

When adding a FeatureGate to Helm charts, define the variable

> `fooEnabled` with a value of `false` in [values.yaml](https://github.com/kubernetes-sigs/service-catalog/blob/master/charts/catalog/values.yaml).  

Reviewer: ```suggestion
`fooEnabled` with the `false` value in [values.yaml](https://github.com/kubernetes-sigs/service-catalog/blob/master/charts/catalog/values.yaml).  
```
